packages/codecs/src/known/runtime.ts

```diff
@@ -18,3 +18,94 @@ export const $MultiAddress = $.Enum({
 });
 
 export type MultiAddress = $.Input<typeof $MultiAddress>;
+
+/**
+ * DispatchError
+ */
+
+export const $TokenError = $.Enum({
```

Comment:
We can use `$.FlatEnum` here if all the values are `null`. You can search around for `$.FlatEnum` in the codebase for references.